registry.json

```
@@ -214,6 +218,10 @@
     "listed": false,
     "version": "4.4.0"
   },
+  "System.Reflection.Metadata" : {
+    "listed": true,
+    "version": "5.0.0"
```

```suggestion
    "version": "1.5.0"
```

So far we have been using the rule that we add the 1st version of netstandard2.0 of the library and upward

---

(the rationale being that if another package that we want to add later has a dependency on an older version, that will be possible to add it easily)

---

Understood - I thought that since the version jump between 1.5.0 and 5.0.0 seems like an intended big gap it makes sense to not support the older one, but I get the dependency reasoning :) Updated!